index.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="UTF-8">
+<meta http-equiv="X-UA-Compatible" content="IE=edge">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300&display=swap" rel="stylesheet">
+<link rel = "stylesheet" href = "style.css" />
+<title>Portfolio</title>
+</head>
+
+<body>
+    
+<div id = "content">
+   
+    <header>
+        
+        <nav id = "menu">
+            <h4 class = "name">Ivan Gaona</h4>
+            <img src = "images/Union.png" alt = "" height = "25">
+        </nav>    
+
+    </header>
+    
+    <section id="headline-section">
+
+       
+        <h2 class = "title">
+        Ivan Gaona
+        </h2>
+    
+        <h3 class = "subtitle">
+        Software Developer    
+        </h3>
+           
+        <p class = "info">
+        I can help you build a product ,feature or website Look through some of my work and experience! 
+        If you like what you see and have a project you need coded, 
+        don’t hestiate to contact me.</p>
+
+    </section>
+    <hr class = "gray">
+    
+    <section id = developer-skills >
+        
+        <ul class = "language">
+            <li class = "purple">Language</li>
+            <li>Javascript</li>
+            <li>Ruby</li>
+            <li>Html</li>
+            <li>Css</li>
+        </ul>
+    
+        <ul class = "frameworks">
+            <li class = "purple">Frameworks</li>
+            <li>Bootstrap</li>
+            <li>Ruby on Rails</li>
+            <li>RSPec</li>
+            <li>CapyBara</li>
+            <li>Selenium</li>
+        </ul>
+    
+        <ul class = "skills">
+            <li class = "purple">Skills</li>
+            <li>Codekit </li>
+            <li>Github</li>         
+            <li>Codepen</li>
+            <li>Gitlab</li>
+            <li>Terminal</li>
+        </ul>
+        
+    </section>
+        <hr class = "gray">
+
+        <main id = "social-media">
+            <a href=""> 
+            <ul class="logos">
+              
+              <li class="github"><img src = "images/Disabled.png" alt = "github-logo"></li>
+              <li><img src = "images/Group.png" alt = "linkedin-logo"></li>
+              <li><img src = "images/Vector.png" alt = "angellist-logo"></li>
+              <li><img src = "images/Twitter.png" alt = "twitter-logo"></li>
+              <li><img src = "images/M.png" alt = "medium-logo"></li>
+              
+            </ul>
+            </a> 
+         
+          <div class = "arrow">
+          <img src = "images/Group55.png" alt = "arrow">
+          <a href="">
+          <h3 class = "work">Work</h3>   
+          </a>
```

Comment:
- [ ] This was a good idea to add the `<a>` tag for the work button, but it could be nicer to wrap even the arrow icon inside the same `<a>` tag since the **work** and the arrow icon should be clickable.